src/gui/entry/EntryURLModel.h

```diff
@@ -43,8 +44,10 @@ class EntryURLModel : public QStandardItemModel
 
 public:
     explicit EntryURLModel(QObject* parent = nullptr);
+
     void setEntryAttributes(EntryAttributes* entryAttributes);
     void insertRow(const QString& key, const QString& value);
+    void setEntryUrl(const QString& entryUrl);
```

Comment:
We pass the entry attributes pointer in the main setup function to this model, why not just pass the entry itself and take both in one shot.

Reply:
`EntryAttributes` will be still needed though? Creating that later from `Entry` doesn't seem to work properly. For simplicity I'd just give the entry URL in `setEntryAttributes()`.

Reply:
I didn't check where this is used so if it won't work no worries.